app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -271,7 +271,6 @@ public void onCreate(Bundle savedInstanceState) {
         // background color needs to be forced
         //TODO: Change themes to use appropriate background colors that don't need overriding.
         TypedValue outValue = new TypedValue();
-        themeContext.getTheme().resolveAttribute(R.attr.messageViewBackgroundColor, outValue, true);
```

Comment:
This only removes half of the code belonging to the feature. We end up with outdated comments and broken code (a background color is set, it just happens to accidentally be Color.TRANSPARENT).

Reply:
yes. I fixed this, in new commits.